### DISCUSSION TITLE
The path in Mobi couldn't find

### DISCUSSION THREAD

**Discussion by Renchugo** [Open-Systems-Pharmacology/Forum#1625] [Questions & Problems] [answered]
Hello, All,

I am new at PK-sim/ Mobi and just want to explore how to apply the available Mobi structure into the new compound. I used the Spatial Structure from: https://github.com/Open-Systems-Pharmacology/Birds-PBKModel

After creating new molecule and sending to Mobi, I created the simulation using Birds-PBKModel spatial structure and new Molecules, however, the error shows: 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/105068944/8e0e6d84-a853-4d8f-a7ab-1ca1a4efdb19)

However, in the mentioned path, the parameter is shown there.
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/105068944/c436ffc0-b04e-451a-a2e6-851a29cea0b3)

how can I deal with this error? is this maybe the problem of "parameter start valus" section?

Thank you in advance!

**Comment by PavelBal** [comment]
`LipoproteinFactorForEgg` is not a default parameter, so you must create it for the new molecule manually. I do not know on how to select a value for this parameter, maybe looking at the already implemented molecules and/or reading the description of the birds model will help.

**Reply by Renchugo** [reply]
Thank you for your help!
Yes, I need to set this parameter manually. I compared with the implemented molecules (RR) in the birds example models and found the only difference is in the 'Parameter Start Values', where _LipoproteinFactorForEgg_ are shown in the example.

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/105068944/50840627-6bdf-4b4c-9a65-eaf932403dc6)

Could you please tell me how can I add these parameters in the model to solve this issue?

Thank you!

**Comment by PavelBal** [comment, marked as answer]
When you click on the "Extend" button in the upper left corner, the entry for the parameter of your new molecule will appear there.

But first, you would have to create this parameter in your molecule.

**Reply by Renchugo** [reply]
Thank you for your help! it fixed.
